### ISSUE TITLE
The training loss in convergence for `llava-Llama-2-13b-chat-hf-pretrain` is larger than 2 [Usage] 

### ISSUE THREAD

**Issue by linhaojia13** [haotian-liu/LLaVA#439]
### Describe the issue

Issue: As shon in this [issue](https://github.com/haotian-liu/LLaVA/issues/62), the training loss in coonvergence should be lower than 2 for `llava-vicuna-chat-hf-pretrain`. However, I run the comnand below for `llava-Llama-2-13b-chat-hf-pretrain` and the training loss in convergence is larger than 2.
Have I do somethining wrong on the training?

Command:
```
#!/bin/bash

# Uncomment and set the following variables correspondingly to run this script:

# MODEL_VERSION=vicuna-v1-3-7b
# MODEL_VERSION=llama-2-7b-chat
MODEL_VERSION=Llama-2-13b-chat-hf

########### DO NOT CHANGE ###########
########### USE THIS FOR BOTH ###########
PROMPT_VERSION=plain
########### DO NOT CHANGE ###########

deepspeed llava/train/train_mem.py \
    --deepspeed ./scripts/zero2.json \
    --model_name_or_path /data/mllm/model_weights/$MODEL_VERSION \
    --version $PROMPT_VERSION \
    --data_path /data/mllm/LLaVA-CC3M-Pretrain-595K/chat.json \
    --image_folder /data/mllm/LLaVA-CC3M-Pretrain-595K/images \
    --vision_tower /data/mllm/model_weights/clip-vit-large-patch14 \
    --tune_mm_mlp_adapter True \
    --mm_vision_select_layer -2 \
    --mm_use_im_start_end False \
    --mm_use_im_patch_token False \
    --bf16 True \
    --output_dir ./checkpoints/llava-$MODEL_VERSION-clip-pretrain \
    --num_train_epochs 1 \
    --per_device_train_batch_size 64 \
    --per_device_eval_batch_size 4 \
    --gradient_accumulation_steps 1 \
    --evaluation_strategy "no" \
    --save_strategy "steps" \
    --save_steps 24000 \
    --save_total_limit 1 \
    --learning_rate 2e-3 \
    --weight_decay 0. \
    --warmup_ratio 0.03 \
    --lr_scheduler_type "cosine" \
    --logging_steps 1 \
    --tf32 True \
    --model_max_length 2048 \
    --gradient_checkpointing True \
    --dataloader_num_workers 4 \
    --lazy_preprocess True \
    --report_to wandb

```

Screenshots:
![图片](https://github.com/haotian-liu/LLaVA/assets/38275612/a860eef3-76e0-4d39-aa30-3d92827d068f)

**Comment by haotian-liu** [comment]
Hi, that figure is for v0 checkpoints. For v1, this is completely normal.
